server/backend/backend.go

```diff
@@ -646,19 +646,77 @@ func (b *Backend) DisableAutoMine() {
 	b.automine = false
 }
 
-func (b *Backend) GetTransactionResultByIndex(context.Context, flowgo.Identifier, uint32) (*access.TransactionResult, error) {
-	// TODO: implement
-	panic("GetTransactionResultByIndex not implemented")
+func (b *Backend) GetTransactionResultByIndex(ctx context.Context, id flowgo.Identifier, index uint32) (*access.TransactionResult, error) {
```

Comment:
why do these use flowgo.Identifier btw? i tried to implement in emulatorGateway in cli here https://github.com/onflow/flow-cli/pull/777/files but it expects the input to be flow.Identifier.